ods_ci/tests/Resources/Page/ModelRegistry/ModelRegistry.resource

```diff
@@ -0,0 +1,400 @@
+    [Arguments]    ${filepath}
+    Open Notebook File In JupyterLab    ${filepath}
+    Open With JupyterLab Menu  Run  Restart Kernel and Run All Cells…
+    Wait Until Page Contains Element    xpath=//div[contains(text(),"Restart") and @class="jp-Dialog-buttonLabel"]
+    Click Element    xpath=//div[contains(text(),"Restart") and @class="jp-Dialog-buttonLabel"]
+    # Somehow misses the check on the active cells without the sleep
+    Sleep    5s
+    Wait Until JupyterLab Code Cell Is Not Active  timeout=120s
+    JupyterLab Code Cell Error Output Should Not Be Visible
+    SeleniumLibrary.Capture Page Screenshot
+
+Wait For Model Registry Containers To Be Ready
+    [Documentation]    Wait for model-registry-deployment to be ready
+    ${result}=    Run Process
+    ...        oc wait --for\=condition\=Available --timeout\=5m -n ${PRJ_TITLE} deployment/model-registry-db
+    ...        shell=true    stderr=STDOUT
+    Log To Console    ${result.stdout}
+    ${result}=    Run Process
+    ...        oc wait --for\=condition\=Available --timeout\=5m -n ${PRJ_TITLE} deployment/model-registry-deployment
+    ...        shell=true    stderr=STDOUT
+    Log To Console    ${result.stdout}
+
+Get Domain
+    [Documentation]  Gets the Domain and returns it to 'Get Cluster Domain And Token'.
+    # Run the command to get the ingress domain
+    ${domain_result}=    Run Process    oc    get    ingresses.config/cluster
+    ...    -o    yaml    stdout=PIPE    stderr=PIPE
+    ${rc}=    Set Variable    ${domain_result.rc}
+    IF    $rc > 0    Fail    Command 'oc whoami -t' returned non-zero exit code: ${rc}
+    ${domain_yaml_output}=    Set Variable    ${domain_result.stdout}
+
+    # Return the domain from stdout
+    ${domain_parsed_yaml}=    Evaluate    yaml.load('''${domain_yaml_output}''', Loader=yaml.FullLoader)
+    ${ingress_domain}=    Set Variable    ${domain_parsed_yaml['spec']['domain']}
+
+    # Return both results
+    RETURN    ${ingress_domain}
+
+Get Token
+    [Documentation]    Gets the Token and returns it to 'Get Cluster Domain And Token'.
+    ${token_result}=    Run Process    oc    whoami    -t    stdout=YES
+    ${rc}=    Set Variable    ${token_result.rc}
+    IF    ${rc} > 0    Fail    Command 'oc whoami -t' returned non-zero exit code: ${rc}
+    ${token}=    Set Variable    ${token_result.stdout}
+    RETURN    ${token}
+
+Apply ServiceMeshMember Configuration
+    [Documentation]    Apply a ServiceMeshMember configuration using oc.
+    Copy File    source=${SERVICE_MESH_MEMBER}    destination=${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml
+    Run    sed -i '' -e 's/<MODEL_REGISTRY_NS>/${NAMESPACE_MODEL_REGISTRY}/' ${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml    # robocop: disable
+    Apply OpenShift Configuration    ${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml
+    Remove File    ${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml
```

Comment:
is this adding the MR namespace into the SMMR? if yes, we have a dedicated kw: `Add Namespace To ServiceMeshMemberRoll`

Reply:
No it's creating a SMM resource in the namespace, which has the effect of also adding the ns to the member roll